test/jdk/java/foreign/TestHandshake.java

```diff
@@ -54,12 +55,13 @@
     static final int ITERATIONS = 5;
     static final int SEGMENT_SIZE = 1_000_000;
     static final int MAX_DELAY_MILLIS = 500;
-    static final int MAX_EXECUTOR_WAIT_SECONDS = 10;
+    static final int MAX_EXECUTOR_WAIT_SECONDS = 20;
```

Comment:
The increased max executor wait time looks good. It will not unnecessarily cause the test to take longer to run, just tolerate slower runs.